oak-store-document/src/test/java/org/apache/jackrabbit/oak/plugins/document/MongoDbTest.java

```diff
@@ -201,7 +201,8 @@ public void updateDocument() {
 
     private static BasicDBObject explain(MongoCollection<BasicDBObject> collection,
                                          Bson query) {
-        return collection.find(query).modifiers(new BasicDBObject("$explain", true)).first();
+        //TODO: Determine if this test is still needed. Modifiers is removed in driver 4, so this has to be reworked
+        return collection.find(query)/*.modifiers(new BasicDBObject("$explain", true))*/.first();
```

Comment:
While those tests are all ignored, this can still be fixed. I think simply calling `collection.find(query).explain(BasicDBObject.class)` will do the trick.